google/auth/transport/aiohttp_req.py

```diff
@@ -0,0 +1,337 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Transport adapter for Async HTTP (aiohttp)."""
+
+from __future__ import absolute_import
+
+import asyncio
+import functools
+import logging
+import numbers
+import time
+
+import aiohttp
+import six
+
+from google.auth import exceptions
+from google.auth import transport
+
+_OAUTH_SCOPES = [
+    "https://www.googleapis.com/auth/appengine.apis",
+    "https://www.googleapis.com/auth/userinfo.email",
+]
+
+_LOGGER = logging.getLogger(__name__)
+
+# Timeout can be re-defined depending on async requirement. Currently made 60s more than
+# sync timeout.
+_DEFAULT_TIMEOUT = 180  # in seconds
+
+
+class _Response(transport.Response):
+    """
+    Requests transport response adapter.
+
+    Args:
+        response (requests.Response): The raw Requests response.
+    """
+
+    def __init__(self, response):
+        self.response = response
```

Comment:
No reason in particular why they are different, just thought it would be easier to implement without the underscores

Reply:
I see the class itself is internal, so either way should be fine, but if we wouldn't want folks relying on this if we made the class public, it may make sense to make the field internal as well.

+1 that these likely should match eachother though

Reply:
Will fix this today.